### ISSUE TITLE
Accessibility Audit: Alt tag for property images is too generic

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#944]
**User Story:** 

As a person using a screen reader, I want the images of the property property to have informative alternative text, so that I understand what is represented in pictures on the site. 

Note: This should apply to the multiple photos experience as well. 

**Additional Detail:** 
Design Asset Required - No

Go to the [Listings](https://homeconnect.detroitmi.gov/listings) page or any [Listing Detail](https://homeconnect.detroitmi.gov/listing/dfce011e-e514-43f1-bced-6cf5cd0386ac/preserve_on_ash_i_3309_14_th_street_detroit_mi) page. The alt text for the main property image is simply "A picture of the building". In order to be less generic, this should be changed to "[property name]" (e.g. "Oak Village Independence"). We should be mindful here of repetition within the screen reader. The screen reader will announce that the focus here is on a photo, we don't need to include that specifically in the alt tag. 

Multiple photos experience should read something like [Property name] image 1, [Property name] image 2, etc. 

**Technical Notes:** 
Change to UIC - This is present on the listing card, but, we should follow the model of Detroit first if the card has not been applied to Detroit yet. 

**AC:** 

When using a screen reader, the Alt text for the image of listing should be read as [property name].
On a multiple photo experience, it should be read as [Property name] image 1, [Property name] image 2, etc. 

**QA Notes:**

**Comment by jnowell** [comment]
This also applies to the property images on the homepage.

**Comment by sarahlazarich** [comment]
Final solution involving multiple photos pending LH input

**Comment by sarahlazarich** [comment]
Lgtm!
